### DISCUSSION TITLE
Two USB adaptor on Synology

### DISCUSSION THREAD

**Discussion by marcochoo77** [bb-qq/r8152#338] [Q&A] [answered]
Hi

Does anyone know how to setup two USB network adaptor on Synology?

I have follow all the install instruction but only 1 USB adaptor is showing up.

I have 2 USB network adaptor plug in but only LAN3 is showing up in Synology network interface. LAN1 and LAN2 is build in 1gbps NIC.

Anyone can help? Thanks.

**Comment by bb-qq** [comment, marked as answer]
This document should help you.
https://github.com/bb-qq/r8152/wiki/Troubleshooting#multiple-identical-devices-do-not-work

**Reply by bb-qq** [reply]
I think I have answered your question, so I will close this thread. If you have any questions, feel free to re-open it.
